### ISSUE TITLE
redesign website UI 

### ISSUE THREAD

**Issue by Parth4git** [openshwprojects/OpenBK7231T_App#952]
it is very basic and not user-friendly. there is no navbar and contact us information available @openshwprojects

**Comment by openshwprojects** [comment]
Hello, which website are you referring to?

**Comment by Parth4git** [comment]
https://openbekeniot.github.io/webapp/devicesList.html

**Comment by openshwprojects** [comment]
Would you like to help?

**Comment by Parth4git** [comment]
Yes  like to help, I will redesign your existing website to keep it relevant, user-friendly, and visually appealing.
Kindly this issue to me . I will like to work on this @openshwprojects

**Comment by openshwprojects** [comment]
That would be great, just remember to also link to https://openbekeniot.github.io/webapp/templateImporter.html 
and please try to consult the ideas and draft with me here
